_data/team-academic.yml

```
@@ -1,20 +1,20 @@
 - name: "Marco"
   surname: "Ciolli"
-  title: "Università di Trento"
+  subtitle: "Università di Trento"
   thumbnailUrl: "Marco Ciolli.jpg"
-  subtitle: "Academic Track committee co-chair"
+  title: "Academic Track committee co-chair"
 - name: "Marco"
   surname: "Minghini"
-  title: "European Commission, Joint Research Centre"
+  subtitle: "European Commission, JRC"
   thumbnailUrl: "Marco Minghini.jpg"
-  subtitle: "Academic Track committee co-chair"
+  title: "Academic Track committee co-chair"
 - name: "Përparim"
   surname: "Ameti"
   thumbnailUrl: "Perparim Ameti.jpg"
   subtitle: "Academic Track committee co-chair"
 - name: "Bashkim"
   surname: "Idrizi"
-  thumbnailUrl: "generic.jpg"
+  thumbnailUrl: "BashkimIdrizi.jpg"
```

Please keep image name consistent: `Firstname Lastname.extension`
```suggestion
  thumbnailUrl: "Bashkim Idrizi.jpg"
```